### PR TITLE
FAQ: remove reference to tox "isolated_build" and improve spelling

### PR DESCRIPTION
**Pull request by deronnax**
`isolated_build` is [the default since tox 4](https://tox.wiki/en/latest/upgrading.html#isolated-environment-on-by-default) Tox 4.0.0 was released on [2022-12-07](https://tox.wiki/en/latest/changelog.html#v4-0-0-2022-12-07).

The grammar changes were suggested by [LanguageTool](https://languagetool.org/).

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -83,8 +83,8 @@ If your package will be used as an application, it might be worth to define an u
 
 ### Is tox supported?
 
-**Yes**. By using the [isolated builds](https://tox.readthedocs.io/en/latest/config.html#conf-isolated_build) `tox` provides,
-you can use it in combination with the PEP 517 compliant build system provided by Poetry.
+**Yes**. Provided that you are using `tox` > 4, you can use it in combination with the PEP 517 compliant build system
```
Comment (radoering) [inline docs/faq.md:86]: Shouldn't it be `tox >= 4`?

We could also add a sentence that you have to set `isolated_build` for tox 3 (with an updated link to https://tox.wiki/en/3.27.1/config.html#conf-isolated_build). No strong feelings about that.

Comment (deronnax) [reply docs/faq.md:86]: OK I will implement both feedbacks. Thank you.

File: docs/faq.md [hunk 2]
```diff
@@ -83,8 +83,8 @@ If your package will be used as an application, it might be worth to define an u
...
+provided by Poetry.
```
Comment (radoering) [inline docs/faq.md:87]:
```suggestion
**Yes**. Provided that you are using `tox` >= 4, you can use it in combination with
the PEP 517 compliant build system provided by Poetry. (With tox 3 you have to set the 
[isolated build](https://tox.wiki/en/3.27.1/config.html#conf-isolated_build) option.)
```

**Comment by radoering** [comment]
pre-commit.ci autofix

**Review by radoering — APPROVED** [review]

**Pull request merged**